mitiq/interface/conversions.py

```diff
@@ -20,6 +20,51 @@ class CircuitConversionError(Exception):
     pass
 
 
+register_to_dict: Dict[Tuple[str, str], Callable[[Any], Circuit]]
+try:
+    register_to_dict
+except NameError:
+    register_to_dict = {}
+
+register_from_dict: Dict[Tuple[str, str], Callable[[Circuit], Any]]
+try:
+    register_from_dict
+except NameError:
+    register_from_dict = {}
+
+
+def register_mitiq_converter(
+    package_name: str,
+    direction: str,
+    convert_function: Union[
+        Callable[[Any], Circuit], Callable[[Circuit], Any]
+    ],
+) -> None:
+    """Registers converters for unsupported circuit types.
+
+    Args:
+        package_name: A quantum circuit module name that is not currently
+            supported by Mitiq. Note: this name should be the same as the
+            return from "circuit".__module__.
+                 See mitiq.SUPPORTED_PROGRAM_TYPES.
+        direction: Specifies the conversion direction relative to the non-Mitiq
+            circuit. i.e. "from" returns a Mitiq/Cirq circuit, "to" returns a
+            Non-Mitiq circuit.
+        convert_function: User specified function to convert to and from an
+            unsupported circuit type. These functions should follow the
+            direction hint above: "from" returns a Mitiq/Cirq circuit,
+            "to" returns a Non-Mitiq circuit.
+    """
+    if direction not in ["to", "from"]:
+        raise ValueError("Invalid direction. Expected 'to' or 'from'.")
+    elif direction == "to":
+        global register_to_dict
+        register_to_dict[(package_name, direction)] = convert_function
```

Comment:
I don't think the `global` keyword is needed here (or in other places), is it? I think this is only needed when the global variable is immutable.